client/src/actions/hanabiActions.js

```diff
@@ -0,0 +1,30 @@
+export function resetGame(){
+  return {type: "RESET_GAME"}
+}
+
+export function addPlayer(playerName){  
+  return {type: "ADD_PLAYER", playerName}
+}
+
+export function startGame(originalPlayers, originalDeck){
+  const deck = [...originalDeck];
+  let cardCount = undefined;
```

Comment:
the default value of an variable with no set value is undefined. I would just set `let cardCount;`